reference/metric.py

```diff
@@ -503,10 +600,19 @@ def main(args):
     )
     parser.add_argument(
         "--score_all",
-        type=bool,
-        default=False,
+        action=argparse.BooleanOptionalAction,
         help="Whether or not to score against all ground truth labels (inclusive of low confidence labels).",
     )
+    parser.add_argument(
+        "--drop_low_detect",
+        action=argparse.BooleanOptionalAction,
+        help="Whether or not to drop predictions that are matched to low confidence labels.",
+    )
+    parser.add_argument(
```

Comment:
Can we provide `costly_dist` as a float? Any value can be used to threshold, and the presence of the argument can be the boolean flag. in `drop_low_confidence_preds`, the kwarg default value can be `None`.

Reply:
Discussed this, will keep binary for now.